hieradata/common/roles/ns.yaml

```diff
@@ -1,17 +1,43 @@
 ---
+include:
+  default:
+    - profile::dns::ns
+    - profile::dns::rndc_key
+    - profile::logging::rsyslog::client
+
+profile::dns::rndc_key::create_admin_key:         true
+profile::dns::rndc_key::create_mdns_key:          true
+profile::dns::rndc_key::host_is_bind_server:      true
+
+# This is the authoritative name server
+profile::dns::ns::authoritative: true
+
+# Any zones should be placed in the "primary zone" dir
+profile::dns::ns::zonedir: 'pz'
+
+# DNS server addresses (used by file templates)
 #profile::dns::ns::mdns_transport_addr: "%{hiera('service__address__mdns')}"
-profile::dns::ns::admin_mgmt_addr:     "%{hiera('netcfg_mgmt_netpart')}.11"
-profile::dns::ns::ns1_mgmt_addr:       "%{hiera('netcfg_mgmt_netpart')}.16"
-profile::dns::ns::ns2_mgmt_addr:       "%{hiera('netcfg_mgmt_netpart')}.17"
-profile::dns::ns::ns1_transport_addr:  "%{hiera('service__address__ns_01')}"
-profile::dns::ns::ns2_transport_addr:  "%{hiera('service__address__ns_02')}"
-profile::dns::ns::ns1_public_addr:     "%{hiera('public__address__ns_01')}"
+profile::dns::ns::admin_mgmt_addr:          "%{hiera('netcfg_mgmt_netpart')}.11"
+profile::dns::ns::ns_mgmt_addr:             "%{hiera('netcfg_mgmt_netpart')}.16"
+profile::dns::ns::resolver1_mgmt_addr:      "%{hiera('netcfg_mgmt_netpart')}.17"
+profile::dns::ns::resolver2_mgmt_addr:      "%{hiera('netcfg_mgmt_netpart')}.18"
+profile::dns::ns::ns_transport_addr:        "%{hiera('service__address__ns')}"
+profile::dns::ns::resolver1_transport_addr: "%{hiera('service__address__resolver_01')}"
+profile::dns::ns::resolver2_transport_addr: "%{hiera('service__address__resolver_02')}"
+profile::dns::ns::ns_public_addr:           "%{hiera('public__address__ns')}"
+profile::dns::ns::ns_public6_addr:          "%{hiera('public6__address__ns')}"
+profile::dns::ns::resolver1_public_addr:    "%{hiera('public__address__resolver_01')}"
+profile::dns::ns::resolver1_public6_addr:   "%{hiera('public6__address__resolver_01')}"
 
+# DNS server firewall configuration
 profile::dns::ns::manage_firewall: true
 profile::dns::ns::firewall_extras:
   source: "%{::network_trp1}/%{::netmask_trp1}"
```

Comment:
Ser at denne blir brukt for port 953 i ns.pp. Skal ikke admin bruke rndc? (admin har kun mgmt)